PYME/LMVis/importTextDialog.py

```diff
@@ -31,22 +31,33 @@ def isnumber(s):
     except:
         return False
     
-
-
-class ImportTextDialog(wx.Dialog):
+class ImportDialog(wx.Dialog):
     requiredVariables = {'x':'x position [nm]',
                         'y':'y position [nm]'}
     recommendedVariables = {'A':'amplitude of Gaussian',
                             't':'time [frames]',
                             'sig':'std. deviation of Gaussian [nm]',
                             'error_x':'fit error in x direction [nm]'}
     niceVariables = {'error_y':'fit error in y direction [nm]'}
-
-    def __init__(self, parent, textFileName):
-        wx.Dialog.__init__(self, parent, title='Import data from text file')
```

Comment:
Really no excuse for not calling wx.Dialog.__init__()

Reply:
You are right. My excuse was, "it works," but it's a sloppy thing to do.

Reply:
Also, I couldn't think of a good way to change the dialog box title at the moment I was coding this, so I thought it'd be easier to leave in each subclassed function. It's actually super easy to change the dialog box title. Not sure where my head was.